### ISSUE TITLE
Please bump dart_console to ^4.0.0

### ISSUE THREAD

**Issue by provokateurin** [leoafarias/fvm#786]
melos ^6.1.0 requires dart_console ^4.0.0 and I would like to update but the constraint on dart_console ^1.2.0 is holding it back.
Unfortunately just bumping the version doesn't work because it then conflicts with the constraint from interact which hasn't seen a release in almost 2 years.

**Comment by leoafarias** [comment]
@provokateurin I would suggest installing FVM standalone through Brew, Chocolatey, or similar options. Is that a possibility?

**Comment by provokateurin** [comment]
Maybe in the future, but regardless of that the dependency should be bumped anyway at some point.

**Comment by leoafarias** [comment]
Yes, eventually, but FVM should be able to be installed globally without any conflict. FVM should not be added as a dev dependency.
